README.md

```diff
@@ -37,6 +37,8 @@ Some packages in otp-ui depend on sibling packages (e.g., `@opentripplanner/core
 2. Run:
    `yarn && yarn dev`
 
+If the Storybook addon bar (a bar of controls at the bottom of the story) does not appear, you may need to clear localStorage by opening the browser console and typing `localStorage.clear()`.
```

Comment:
For me I had to run `yarn clean`

Reply:
storybook bug that's been around for years